README.md

```diff
@@ -7,35 +7,49 @@
 - Manage proxy admin rights.
 - Easily use in tests.
 
-## Quickstart
+## Installation
+
+### Buidler
 
-Install the plugin for your favorite toolchain:
 ```
-npm install --save-dev @openzeppelin/buidler-upgrades
+npm install --save-dev @openzeppelin/buidler-upgrades @nomiclabs/buidler-ethers ethers
 ```
+
+This installs our Buidler plugin along with the necessary peer dependencies.
+
+### Truffle
+
 ```
 npm install --save-dev @openzeppelin/truffle-upgrades
 ```
 
-And set up a [Buidler script](https://buidler.dev/guides/scripts.html) to deploy and upgrade, after adding `usePlugin('@openzeppelin/buidler-upgrades')` to your `buidler.config.js`:
+## Usage
+
+See the documentation for each plugin, or take a look at the sample code snippets below.
+
+| [<img src="assets/buidler.svg" height="20px" width="30px" alt="">Buidler](./packages/plugin-buidler/README.md)| [<img src="assets/truffle.svg" height="20px" width="30px" alt="">Truffle](./packages/plugin-truffle/README.md) |
+|-|-|
+
+Buidler users will be able to write [scripts](https://buidler.dev/guides/scripts.html) that use the plugin to deploy or upgrade a contract.
```

Comment:
Although it's not the idea to showcase the whole API, I think we're rendering the `admin` module invisible. What do you think of using at least one its functions?

Reply:
I extended this sentence to mention admin, but didn't add it to the snippet to keep it smaller. The admin module is documented in detail in each plugin.